src/components/side_panel/chart/pie_chart/pie_chart_config_panel.ts

```diff
@@ -0,0 +1,24 @@
+import { createRange } from "../../../../helpers";
+import { LineBarPieConfigPanel } from "../line_bar_pie_panel/config_panel";
+
+export class PieConfigPanel extends LineBarPieConfigPanel {
+  static template = "o-spreadsheet-PieConfigPanel";
+
+  doesDataSetContainNegativeValues() {
+    let isNegative = false;
```

Comment:
You were 100% correct, using runtime was not feasible since we already filtered the values. I just discussed this with François, we think that the spec could be updated to show the message as long as there is bnon positive (0 or negative) values. Based on that I think that we could just compare the dataset length between runtime and definition. if there is a discrepency ,it means that we filtered some non-positive values.